src/index.js

```diff
@@ -3,11 +3,14 @@ import ReactDOM from 'react-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import { HashRouter } from 'react-router-dom';
+import { AuthProvider } from './context/AuthContext';
 ReactDOM.render(
   <React.StrictMode>
-    <HashRouter>
-      <App />
-    </HashRouter>
+    <AuthProvider>
+      <HashRouter>
+        <App />
+      </HashRouter>
```

Comment:
I understand the use of a hash router now, that it's specific to electron.js. This is fantastic! Awesome job Joseph :)